packages/server/test/util/testUtils.ts

```diff
@@ -82,3 +85,15 @@ export function setupIntegrationTest(
     return agent;
   };
 }
+
+const notifMock = mocked({ notifyUser: jest.fn() }, true);
+/**
+ * Module Modifier tests can pass to setupIntegrationTest to mock the notifService and to expect things
+ * ex:
+ * const supertest = setupIntegrationTest(QuestionModule, modifyMockNotifs);
+ */
+
+export const modifyMockNotifs: ModuleModifier = (t) =>
```

Comment:
ngl really proud of this code lmao. 
See the question integration test for usage. It let's us check whether a user has been notified without being a huge pain